client/src/components/ui/SelectDropDown.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
```

Comment:
the file should be SelectDropdown

Reply:
drop down is two words. the other file is the wrong one

Reply:
well you named the class SelectDropdown and I like Dropdown better and if you google it it's hyphenated so I think we decide.